web/assets/js/views/DeceasedReport.js

```diff
@@ -0,0 +1,13 @@
+$(document).ready(function () {
+    $('input:radio').on('change', function () {
```

Comment:
Instead of listening for any radio button change, can we add something to limit the scope here? Maybe add an id to row div and target input:radio inside that div?